pkgs/data/themes/gruvbox-gtk-theme/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
```

```suggestion
, stdenvNoCC
```
No need to use a stdenv with a full blown compilation system.